DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/Utils/SnapshotTestCase.swift

```diff
@@ -11,8 +11,11 @@ import TestUtilities
 @testable import DatadogSessionReplay
 @testable import SRHost
 
-private var defaultPrivacyLevel: SessionReplayPrivacyLevel {
-    return SessionReplay.Configuration(replaySampleRate: 100).defaultPrivacyLevel
+private var defaultTextAndInputPrivacyLevel: TextAndInputPrivacyLevel {
+    return .maskSensitiveInputs
+}
+internal var allTextAndInputPrivacyLevels: [TextAndInputPrivacyLevel] {
+    return TextAndInputPrivacyLevel.allCases
 }
```

Comment:
It's used in [SRSnapshotTests](https://github.com/DataDog/dd-sdk-ios/blob/b2086fa3290905f56ce10df80b03268f84963d70/DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/SRSnapshotTests.swift#L12), so I would need to import `DDSessionReplay` there otherwise. That is why `defaultImagePrivacyLevel` and `defaultTextAndInputPrivacyLevel` are also defined in this file.